modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc

```diff
@@ -11,11 +11,12 @@ You can use the `prometheus-adapter` resource to expose custom application metri
 
 .Prerequisites
 
-* You have a custom Prometheus instance installed. In this example, it is presumed that Prometheus was installed in a user-defined `custom-prometheus` project.
+* You have a custom Prometheus instance installed as a Prometheus pod managed by a deployment or statefulset but not by a Prometheus custom resource (CR).  
+* You have installed the custom Prometheus instance in a user-defined `custom-prometheus` project.
 +
-[NOTE]
+[IMPORTANT]
 ====
-Custom Prometheus instances and the Prometheus Operator installed through Operator Lifecycle Manager (OLM) can cause issues with user-defined workload monitoring if it is enabled. Custom Prometheus instances are not supported in {product-title}.
+Custom Prometheus instances and the Prometheus Operator installed through Operator Lifecycle Manager (OLM) are not compatible with user-defined monitoring if user-defined monitoring is enabled. Therefore, custom Prometheus instances that are installed as a Prometheus custom resource (CR) managed by the OLM Prometheus Operator are not supported in {product-title}.
```

Comment:
```suggestion
Custom Prometheus instances and the Prometheus Operator installed through Operator Lifecycle Manager (OLM) are not compatible with user-defined monitoring, if user-defined monitoring is enabled. Therefore, custom Prometheus instances that are installed as a Prometheus custom resource (CR) managed by the OLM Prometheus Operator are not supported in {product-title}.
```

Reply:
Maybe add a comma here, to make parsing the sentence easier?

Reply:
I've revised it to put the "if" clause at the beginning to make parsing the meaning easier.

Reply:
I don't see this change in the file?